MapboxVision/Services/Recording/RecordCoordinator.swift

```diff
@@ -33,7 +33,14 @@ final class RecordCoordinator {
     }
 
     private var trimRequestCache = [Int: [VideoTrimRequest]]()
-    private(set) var isRecording: Bool = false
+    var isRecording: Bool {
```

Comment:
nit: Can we simplify it to:

```suggestion
    var isRecording: Bool {
        return processingQueue.sync {
            _isRecording
        }
    }
```

?

Reply:
nice, didn't know swift can do that, will change

Reply:
done